src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -2493,7 +2493,7 @@ action&lt;GridDefinitionId>GridDefinitionModifier
     
 action&lt;GridDefinitionId>GridQueryBuilderModifier
```

Comment:
Hi, these hooks are available since 1.7.5 as far as I know 🤔 
Look at https://github.com/PrestaShop/PrestaShop/blob/1.7.5.x/src/Core/Grid/Data/Factory/DoctrineGridDataFactory.php#L90 for example

Reply:
Oh! you are right!
I will correct my PR, but I noticed that the hooks were registered in the "hooks" table only since version 1.7.6 as you can see at https://github.com/PrestaShop/PrestaShop/blob/1.7.6.x/install-dev/upgrade/sql/1.7.6.0.sql

Reply:
After checking the other hooks, this pull request can be canceled because all are available since 1.7.5.

Reply:
No worries, thanks for the PR it was a good intent 😉